### DISCUSSION TITLE
Mouse and keyboard stop responding, must force reboot from CAD screen

### DISCUSSION THREAD

**Discussion by rtuite** [debauchee/barrier#1448] [General]
I'm not a regular user of GitHub so I'm not sure if I'm searching incorrectly or this hasn't been reported yet, but I can't seem to find mention of this. I'm also trying to keep with the etiquette of this group, but I'm unfamiliar so I apologize in advance if I'm not meeting expectations. 

I've had this happen twice and can't reproduce it reliably. I've been using this setup for about a week, and this happened yesterday and again today.

Configuration: 
Two desktop PCs running Windows 10, latest updates on both. Each machine has 3 monitors in a row, one row above the other (so, 6 monitors in a 3x2 configuration). The main machine, configured as the server, is a newer Ryzen 7 3700x w/ a 1660 Super GPU. The client is an old Xeon E3-1226 v3 with a Quadro K4000 GPU.

Issue:
While the mouse is on the top screen, it will hang for a moment then disappear. I can no longer input on either system using the keyboard and mouse connected to the server. If I hit Ctrl+Alt+Del I can use the mouse and keyboard. If I return to Windows (such as by launching task manager) I lose input again. If I reboot from the C+A+D screen, the system reboots and functions normally. 

Attempts to troubleshoot while unable to use kbd/mouse normally:
- Plug different kbd/mouse into server - no effect
- Plug kbd/mouse into client - Can control client normally
- Restart Barrier on client with other kbd/mouse - no effect
- Remote into server with another system - Cannot control server, no reaction to keboard or mouse input from remote system
- Ctrl+Alt+Del - Can interact with system normally on windows security screen
- Exit security screen - Issue returns
- Reboot from security screen - System goes back to normal after reboot 

I'm guessing that Barrier is crashing on my server while working on the client, but I can't be sure. Should I file this as a bug report or is this a known issue? If the latter, is there a fix that I can't find?

**Comment by MandersonMuahahaha** [comment]
I was having an issue similar to this - in my case, I found that it was because on one of the systems, I had an image loaded in my clipboard buffer (copied an image with intent to paste it) - Barrier didn't play nice with this, and was barfing when trying to switch to the next window. I just learned this and haven't yet experimented with different types of images in the copy buffer. Workaround that worked for me was to simply copy text into the clipboard buffer, then Barrier "came back to life."

Another variable I have in the mix is that I use Ditto clipboard manager, which could be compounding the issue. In any case, I think I've figured out my dilemma for now. Hope this helps you.
